files/default/test-neutron-ha-tool.py

```diff
@@ -102,22 +102,23 @@ class TestL3AgentMigrate(unittest.TestCase):
     def test_no_dead_agents_returns_zero(self):
         neutron_client = make_neturon_client(live_agents=2)
 
-        result = ha_tool.l3_agent_migrate(neutron_client)
+        result = ha_tool.l3_agent_migrate(neutron_client, None)
 
         self.assertEqual(0, result)
 
     def test_no_alive_agents_returns_one(self):
```

Comment:
I know this is separate from this PR, but I just noticed that `_returns_one` is cryptic - it should describe what the `1` value signifies rather than just the value itself.